data/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'org.jetbrains.kotlin.android.extensions'
+android {
+    compileSdkVersion 26
+    buildToolsVersion "26.0.1"
+
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 26
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
got time to refactor it.

1 step:
create file in the main app gradle folder (libraries.gradle) look like:
class Libraries extends Expando {}

def supportLibraryVersion = "25.3.1"
def googlePlayServicesVersion = "11.0.1"

project.ext.libraries = new Libraries()
def libraries = project.ext.libraries

libraries.supportAnnotations = "com.android.support:support-annotations:$supportLibraryVersion"
libraries.design = "com.android.support:design:$supportLibraryVersion"
libraries.recyclerView = "com.android.support:recyclerview-v7:$supportLibraryVersion"

2.step
On module gradle add on dependencies{}
 def libs = project.libraries
and then use defined values like
    androidTestCompile libs.supportAnnotations
    androidTestCompile libs.design

    compile libs.supportAnnotations
    compile libs.design